app/views/transactions/_form.html.slim

```diff
@@ -1,41 +1,53 @@
 = simple_form_for(transaction, remote: true, html: { class: 'transaction' }) do |f|
   = f.error_notification
-  .col-sm-9
-    .col-md-3.col-sm-6.col-xs-6
-      = f.input :amount, label: false,
-        input_html: { value: f.object.amount == 0 ? nil : f.object.amount }
-    .col-md-3.col-sm-6.col-xs-6
-      - if f.object.category.try(:system?)
-        = f.input :category, label: false, disabled: true
-      - else
-        = f.association :category, as: :grouped_select,
-          collection: current_organization.categories.grouped_by_type,
-          group_method: :last, prompt: 'Category', label: false, value_method: :id,
-          selected: @q.nil? ? f.object.category_id \
-            : (@q.category_id_eq && !Category.find(@q.category_id_eq).system? \
-            ? @q.category_id_eq : nil)
+  .col-sm-12
+    .row
+      .col-md-6.col-sm-12.col-xs-12
```

Comment:
@yunixon don't use custom layout. Use normal bootstrap horizontal form. And return back labels.